### PR TITLE
[Staging] Migrate to v0.2.0 version of the charts scripts

### PR DESCRIPTION
**Pull request by aiyengar2**
Related Issue: https://github.com/rancher/rancher/issues/32199

### CODE REVIEW

**Comment by cbron** [comment]
How does this relate to #1079

**Comment by aiyengar2** [comment]
> How does this relate to #1079

Unrelated. That needs to be done in a separate PR.

This PR just removes the existing rebase functionality, which prints out the drift between the current chart base and a selected upstream.

**Comment by aiyengar2** [comment]
Should be up to date if the last commit to dev-v2.5-source is 185bea3bcac08604c0cb077faed5877e793108d9

File: scripts/version [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -e
 
-CHARTS_BUILD_SCRIPT_VERSION=v0.0.4
+CHARTS_BUILD_SCRIPTS_REPO=https://github.com/aiyengar2/charts-build-scripts.git
+CHARTS_BUILD_SCRIPT_VERSION=v0.2.0
```
Comment (aiyengar2) [inline scripts/version]: The only change from this commit that will be left behind is this one line

**Comment by aiyengar2** [comment]
Will need to revert the temp commit in favor of the one-line change once the charts-build-script changes are merged in.

**Comment by aiyengar2** [comment]
Ran `make validate && ./scripts/package-ci` successfully.

```bash
... a TON of logged lines ...
INFO[0068] Successfully validated against release-v2.5! 
fleet passed ci
fleet-agent passed ci
fleet-crd passed ci
longhorn passed ci
rancher-alerting-drivers passed ci
rancher-backup passed ci
rancher-backup-crd passed ci
rancher-cis-benchmark passed ci
rancher-external-ip-webhook passed ci
rancher-gatekeeper passed ci
rancher-grafana passed ci
rancher-istio-1.8 passed ci
rancher-istio-1.9 passed ci
rancher-kiali-server passed ci
rancher-kube-state-metrics passed ci
rancher-logging passed ci
rancher-monitoring passed ci
rancher-node-exporter passed ci
rancher-operator passed ci
rancher-operator-crd passed ci
rancher-prom2teams passed ci
rancher-prometheus-adapter passed ci
rancher-pushprox passed ci
rancher-sachet passed ci
rancher-tracing passed ci
rancher-vsphere-cpi passed ci
rancher-vsphere-csi passed ci
rancher-webhook passed ci
rancher-windows-exporter passed ci
rancher-wins-upgrader passed ci
rio passed ci

All packages pass ci!
```

**Review by cbron — APPROVED** [review]
Not reviewing this, if it fails we can revert.

**Pull request merged**